### PR TITLE
Colorize Logger

### PR DESCRIPTION
**Pull request by vg3095**
This PR is in reference to issue #100 .
It adds the ability to colorize logs output.
Make sure before testing it , you have installed coloredlogs

`pip install coloredlogs`

I have tested it while generating a Simulation object from the QuickStart Guide.
Here is a screenshot attached for the same.

colored_logs

### CODE REVIEW

**Comment by wkerzendorf** [comment]
@vg3095 please solve this problem without added dependencies and so many changes in so many files.

**Comment by vg3095** [comment]
I have used a script directly from this stackoverflow thread [#Link](http://stackoverflow.com/questions/384076/how-can-i-color-python-logging-output) 
Screenshot
color_2



I was mistaken to import coloredlogs in every file before. It will work , if we add it to only tardis/__init__.py . (Using coloredlogs module). 
If additional formatting for log output is required, the use of coloredlogs module will simplify it.

File: tardis/color.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import logging
```
Comment (wkerzendorf) [inline tardis/color.py]: this should go  in a util package.

File: tardis/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 import logging
-
+import color
```
Comment (wkerzendorf) [inline tardis/__init__.py]: always use absolute imports

File: tardis/color.py (hunk 1 above)
Comment (yeganer) [inline tardis/color.py]: As this code is borrowed from stackoverflow, you should mention the source.

**Comment by vg3095** [comment]
Updated with proposed changes.

**Comment by vg3095** [comment]
@yeganer 
From the official documentation . [(Link)](https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles)
1. > When logging was added to the Python standard library, the only way of formatting messages with variable content was to use the %-formatting method. Since then, Python has gained two new formatting approaches: string.Template (added in Python 2.4) and str.format() (added in Python 2.6).

2. > **_Logging(as of 3.2)_** provides improved support for these two additional formatting styles. 

3. >you cannot directly make logging calls using str.format() or string.Template syntax, because internally the logging package uses %-formatting to merge the format string and the variable arguments. There would no changing this while preserving backward compatibility, since all logging calls which are out there in existing code will be using %-format strings.

So, inshort , this means Logging module supports newer string formatting from Python 3.2 onwards.

**Comment by wkerzendorf** [comment]
@vg3095 Should this be merged?

**Comment by vg3095** [comment]
@wkerzendorf According to me , it can be merged.

**Pull request merged**